src/main/java/com/conveyal/gtfs/api/graphql/PatternFetcher.java

```diff
@@ -15,14 +15,51 @@
  * Created by matthewc on 3/9/16.
  */
 public class PatternFetcher {
+    public static List<WrappedGTFSEntity<Pattern>> apex(DataFetchingEnvironment env) {
+        Collection<FeedSource> feeds;
+
+        List<String> feedId = (List<String>) env.getArgument("feed_id");
+        feeds = feedId.stream().map(ApiMain::getFeedSource).collect(Collectors.toList());
+
+        List<WrappedGTFSEntity<Pattern>> patterns = new ArrayList<>();
+
+        for (FeedSource feed : feeds) {
```

Comment:
I think for this (and the similar issues below) I just followed your lead in the StopFetcher class: https://github.com/conveyal/gtfs-api/blob/4bdb98fbf6498d4bb12a5930c7ac77489b262b17/src/main/java/com/conveyal/gtfs/api/graphql/StopFetcher.java#L41-L56

So, I guess we need to fix all of these classes...